exporter/coralogixexporter/go.mod

```
@@ -71,3 +70,6 @@ retract (
 	v0.76.1
 	v0.65.0
 )
+
+// ambiguous import: found package cloud.google.com/go/compute/metadata in multiple modules
+replace cloud.google.com/go => cloud.google.com/go v0.110.2
```

We may want to replace to the latest version https://pkg.go.dev/cloud.google.com/go?tab=versions
```suggestion
replace cloud.google.com/go => cloud.google.com/go v0.110.7
```